loadtests/research/locustfile.py

```diff
@@ -0,0 +1,77 @@
+import requests
```

Comment:
Please reorder imports according to the pep8.

```suggestion
import os

import requests
from locust.core import TaskSet, task, HttpLocust
```